docs/sql-ref-syntax-qry-select-inline-table.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: global
+title: Inline Table
+displayTitle: Inline Table
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+An inline table is a temporary table created using a VALUES clause.
+
+### Syntax
+
+{% highlight sql %}
+VALUES ( expression [ , ... ] ) [ [ AS ] table_alias ]
+{% endhighlight %}
+
+### Parameters
+
+<dl>
+  <dt><code><em>expression</em></code></dt>
+  <dd>
+    Specifies a combination of one or more values, operators and SQL functions that results in a value.
+  </dd>
+</dl>
+<dl>
+  <dt><code><em>table_alias</em></code></dt>
+  <dd>
+    Specifies a temporary name given to a table.
```

Comment:
Could you write more details for `table_alias`? For example, the case `tableName(col1, col2, ...)`.